server.go

```diff
@@ -172,18 +187,21 @@ Current settings:
 				}
 
 			default:
-				msg = fmt.Sprintf("Sorry, I couldn't process that request: %s", req.Event.Text)
+				subject = fmt.Sprintf("Sorry, I couldn't process that request: %s", req.Event.Text)
```

Comment:
you don't appear to actually send the `subject` text to slack at any point

Reply:
oops, pivoted to send `subject` here with 6474df5 -> f77a5bd.